client/src/dialogs/documentswindow.cpp

```diff
@@ -186,6 +187,11 @@ void DocumentsWindow::loadDocumentsFor(const QString &id, LinkedItemType itemTyp
     }
 }
 
+void DocumentsWindow::setCreateNewEnabled(bool enabled)
+{
+    ui->attachButton->setEnabled(enabled);
```

Comment:
I like when disabled widgets give a hint about why they're disabled, so that the user can know what to do to enable it.
In this case this would mean something like this:

```
    if (!enabled)
        ui->attachButton->setToolTip(i18n("Attaching new documents requires being online"));
   else
        ui->attachButton->setToolTip(QString());
```